expression/chunk_executor.go

```diff
@@ -64,6 +64,100 @@ func VectorizedExecute(ctx sessionctx.Context, exprs []Expression, iterator *chu
 	return nil
 }
 
+func evalOneVec(ctx sessionctx.Context, expr Expression, input *chunk.Chunk, output *chunk.Chunk, colIdx int) error {
+	ft := expr.GetType()
+	result := output.GetOrNewColumn(colIdx, ft, input.NumRows())
```

Comment:
I think we should raise an error when we can not get a column from `output`. Because in this situation, the schema of the `projection` operator is incorrect.

Reply:
Do you mean that it's the caller's duty to initialize the output Chunk rightly?

Reply:
exactly.

Reply:
Updated.
Now the method `GetOrNewColumn` is removed and is replaced with `Chunk.Column`.